### PR TITLE
[READY] Adds an new Selectable Organs plus new traits.

### PR DESCRIPTION
**Pull request by Guidesu**
## About The Pull Request
This would add more organs type to you select from the round start.
And just some traits which will impact the game but not total gaming changing.

Brawler
Basic training in Fireman carry
High Endurance
High Pain Tolerance
Limbs Endurance and Enduranced Hands are new positives traits that WILL impact the game.

Brawler makes you a little better in unarmed combat
Basic training in Fireman Carry or BTFC will make you gain basic fireman carrying skills but NOT BETTER THAN THE PARAMEDIC.
High Endurance negates slowdown from being wounded but it DOES NOT negate crit and you still feeling pain so roleplay correctly, this DO NOT makes you invincible just makes you are little more resilient.
High Pain Tolerance negates SOFT CRIT, not HARD crit. **ONLY SOFT CRIT**, you still have a fuckton of slowdown and are mostly fucked up and at this point, you are fighting for your life (like CM tbh. Soft crit does not make you handicapped but that scales with pain and we do not have pain).
Limbs Endurance makes you more resistant to wounds but DOES NOT NEGATE them.
Enduranced hands just make you be able to take LAMPS without harming your hands I think.

Also adds some traits from old-base which makes you lil dumb and some others negative silly traits

//Demonstration:  Soft Crit
image

//Hard Crit with Pain Tolerance trait
image

## Why It's Good For The Game
More customization is always good

## Changelog
:cl:
add: Several new traits and a few organs to the menu! Check it out!
/:cl:

### CODE REVIEW

**Comment by Stalkeros2** [comment]
What the fuck.
No like really what.
Uh.

**Comment by Guidesu** [comment]
> What the fuck.
> No like really what.
> Uh.

Funny.

To clarify: Talking about balance on the Github in a purely objective way counts as technical analysis. Furthermore, if the person behind the PR invites discussion, it is fine as long as it does not fall into a series of comments mimicking an editorial. Keep it clean, technical, and objective.

Please refrain from using Github as an editorial. It is not meant for expressing your opinion on gameplay. Unless it is for code review or technical analysis, please refrain from commenting, as we have seen multitudes of people abusing the Github and creating unncessary drama and hate against other players, not to mention it makes it harder to find relevant comments that are pertinent to what Github is for. In fact, please be aware that people outside of our community often participate in the comments to drive a wedge between players. Not everyone has good intentions.

**Comment by Guidesu** [comment]
> What the fuck.
> No like really what.
> Uh.

So or your shut up or you suggest something useful.
If not go to discord dumbass i am not into your bullshit.

**Comment by Guidesu** [comment]
Bleh this sound annoying i will just make synth antro to be able to pick organs and use this PR to adds new organs

**Comment by Guidesu** [comment]
> Would you mind actually filling out the fields for the PR so we know what this is?
Yes my laziness mind

**Comment by YakumoChen** [comment]
Extreme no from me.

Combat advantage quirks are _explicitly_ disallowed by the system, powergaming is the only result from adding the positive quirks.

**Comment by Guidesu** [comment]
> Extreme no from me.
> 
> Combat advantage quirks are _explicitly_ disallowed by the system, powergaming is the only result from adding the positive quirks.

Explain how resisting more in crit is powergaming my friend

**Comment by Guidesu** [comment]
Powergaming in SS13 is basically trying to get power in the forms of weapons, gear, items and access over all else including RP(if applicable) and the enjoyment of others(always applicable). What's considered powergaming can vary from server to server but is generally more frowned upon the higher the  RP level.

**Comment by Guidesu** [comment]
Powergaming is defined as: going out of your way to give yourself an extreme mechanical edge over the rest of the crew. Do not give yourself all-access, a combat hardsuit, several laser weapons, and a squad of E-Magged borgs when facing off against a skeleton crew as Traitor. This goes against the aims and goals of the server which is to provide a roleplaying environment where action and conflict is plentiful.

**Comment by Guidesu** [comment]
Did you even saw Skyrat rules about power gaming?

**Comment by Guidesu** [comment]
**_holy macaroni please read the rules before talking with me about powergaming. _**

**Comment by YakumoChen** [comment]
Yea
Powergaming with quirks is not allowed. The system is not designed for quirks that give (mostly invisible to other players) combat advantages

Furthermore some of these traits are covered by the Skillchip system and should not overlap

**Comment by Guidesu** [comment]
> Yea
> Powergaming with quirks is not allowed. The system is not designed for quirks that give (mostly invisible to other players) combat advantages
> 
> Furthermore some of these traits are covered by the Skillchip system and should not overlap

wtf are you even talking.
Did you saw what POWERGAMING MEANS??

**Comment by YakumoChen** [comment]
Yes. Players will simply take quirks that give minor disadvantages to take the quirks with the best possible advantages. 

Positive quirks should only be flavor or minor, non-combat benefits.

Negative quirks are disadvantages of any kind because they're designed to be taken as a challenge

**Comment by Guidesu** [comment]
> Yes. Players will simply take quirks that give minor disadvantages to take the quirks with the best possible advantages.
> 
> Positive quirks should only be flavor or minor, non-combat benefits.
> 
> Negative quirks are disadvantages of any kind because they're designed to be taken as a challenge

e

**Comment by Guidesu** [comment]
I kinda knew it someone would talk about power gaming and shit.
So I don't care I will wait for an maintainer official word

**Comment by hugmeorelse** [comment]
This conversation is closed due to toxicity from multiple parties. Punishments will be dished out.

File: code/modules/mob/living/blood.dm [hunk 1]
```diff
@@ -85,6 +85,11 @@
 	if(!(NOBLOOD in dna.species.species_traits))
 		..()
 
+//skyrat edit - hemophilia quirk
```
Comment (Funce) [inline code/modules/mob/living/blood.dm:88]: Indent your comment

File: code/__DEFINES/traits.dm [hunk 2]
```diff
@@ -640,3 +640,7 @@ Remember to update _globalvars/traits.dm if you're adding/removing/renaming trai
 #define STATION_TRAIT_FILLED_MAINT "station_trait_filled_maint"
 #define STATION_TRAIT_EMPTY_MAINT "station_trait_empty_maint"
 #define STATION_TRAIT_PDA_GLITCHED "station_trait_pda_glitched"
+
+
+//skyrat_traits
```
Comment (Funce) [inline code/__DEFINES/traits.dm:649]: Skyrat traits go in file `code\__DEFINES\~skyrat_defines\traits.dm`

**Review by Funce — CHANGES_REQUESTED** [review]
You've got an extra file `organs` for some reason. Easy to miss.

Some of these descriptions need to be checked over for grammar, get someone to help you with that. 

Otherwise, code's fine, excepting the below issues. It's just balance here that needs to be checked. @hugmeorelse

**Comment by Gandalf2k15** [comment]
Hello. Please address the issues with this PR and consider reopening later.

**Pull request closed without merging**